internal/cli/organizations.go

```diff
@@ -611,8 +626,8 @@ func listMembersRolesOrganizationCmd(cli *cli) *cobra.Command {
 		Args:  cobra.MaximumNArgs(1),
 		Short: "List organization members for a role",
 		Long:  "List organization members that have a given role assigned to them.",
-		Example: `auth0 orgs roles members list
-auth0 orgs roles members list <org id> --role-id role`,
+		Example: `  auth0 orgs roles members list
+  auth0 orgs roles members list <org id> --role-id role`,
```

Comment:
Consider adding in a future PR an example with the `ls` alias, like in other `list` commands.

Reply:
Also, as an org is likely to have many members that have a particular role, consider in a future PR adding an example with the `-n` flag, like in other similar commands.

Reply:
Consider as well adding an example with the `--json` flag, like other similar commands.